omniduct/databases/base.py

```diff
@@ -254,8 +254,6 @@ def push(self, df, table, if_exists='fail', **kwargs):
             The pandas dataframe to push into the data store.
         table : str
             The name of the table into which the dataframe should be pushed.
-        overwrite : bool
```

Comment:
Documentation here leaves a lot to be desired (my fault!). I've been focussing on functionality and structure before documenting everything... but I'm getting pretty happy with the structure, so next up, documentation!

On that note, thanks for improving this. But if you are removing the `overwrite` documentation, can we replace it with the `if_exists` documentation, which is lifted from the `DataFrame.to_sql` method. Basically: `if_exists : {‘fail’, ‘replace’, ‘append’}, default ‘fail’`.